app/presenters/keyword_presenter.rb

```diff
@@ -14,11 +14,19 @@ def initialize(keyword:)
     @keyword = keyword
   end
 
+  def status
+    keyword.status.capitalize
```

Comment:
Just a small thing, should we localize the status :)

Reply:
Fixed in 9cdf12a0d27bd886e7cc3bd5248a02835b4523b0